lang/en.json

```diff
@@ -544,6 +544,14 @@
     "spendPoints": "Spend Points",
     "spendPrestige": "Spend Prestige",
     "spouse": "Spouse",
+    "sibling": "Sibling",
+    "aunt": "Aunt",
+    "uncle": "Uncle",
+    "cousin": "Cousin",
+    "father": "Father",
+    "mother": "Mother",
```

Comment:
We want to keep the keys in here sorted alphabetically.  That said, I'd be happy to group these together under a "relationship" key.